code/modules/customitems/item_defines.dm

```diff
@@ -1,753 +0,0 @@
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "asher_spock_1"
-	amount_per_transfer_from_this = 5
-	volume = 15
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/New()
-	..()
-	reagents.remove_reagent(TRICORDRAZINE, 30)
-	reagents.add_reagent(OXYCODONE, 15)
-	update_icon()
-	return
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack_self(mob/user as mob)
-	to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack(mob/M as mob, mob/user as mob)
-	if (user.ckey != "nerezza") //Because this can end up in the wrong hands, let's make it useless for them!
-		to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
-		return
-	if(!reagents.total_volume)
-		to_chat(user, "<span class='warning'>\The [src] is empty.</span>")
-		return
-	if (!( istype(M, /mob) ))
-		return
-	if (reagents.total_volume)
-		if (M == user && user.ckey == "nerezza") //Make sure this is being used by the right person, for the right reason (self injection)
-			visible_message("<span class='notice'>[user] presses their </span>\
-				penlight against their skin, quickly clicking the button once.", \
-				"<span class='notice'>You press the disguised autoinjector against your skin and click the button. There's a sharp pain at the injection site that rapidly fades.</span>", \
-				"You hear a rustle as someone moves nearby, then a sharp click.")
-		if (M != user && user.ckey == "nerezza") //Woah now, you better be careful partner
-			to_chat(user, "<span class='notice'>You don't want to contaminate the autoinjector.</span>")
-			return
-		src.reagents.reaction(M, INGEST)
-		if(M.reagents)
-			var/trans = reagents.trans_to(M, amount_per_transfer_from_this)
-			to_chat(user, "<span class='notice'>[trans] units injected. [reagents.total_volume] units remaining in \the [src].</span>")
-	return
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/examine(mob/user as mob)
-	..()
-	if(user.ckey != "nerezza")
-		return //Only the owner knows how to examine the contents.
-	if(reagents && reagents.reagent_list.len)
-		for(var/datum/reagent/R in reagents.reagent_list)
-			to_chat(usr, "<span class='notice'>You examine the penlight closely and see that it has [R.volume] units of [R.name] stored.</span>")
-	else
-		to_chat(usr, "<span class='notice'>You examine the penlight closely and see that it is currently empty.</span>")
-
-//End strange penlight
-
-/obj/item/weapon/card/id/fluff/asher_spock_2 //Nerezza: Asher Spock
-	name = "Odysses Specialist ID card"
-	desc = "A special identification card with a red cross signifying an emergency physician has specialised in Odysseus operations and maintenance.\nIt grants the owner recharge bay access."
```

Comment:
1. It claims to do something stupid
2. It doesn't actually do it
3. There's a fucking typo in the name